### DISCUSSION TITLE
test reliability?

### DISCUSSION THREAD

**Discussion by dd-olive** [bluesky-social/atproto#2437] [Dev Debugging]
I'm just getting a dev environment set up in the hopes that I can be helpful, but my tests aren't passing out of the box. I have a working build, including a working docker installation, but it looks like the ozone test "blob-divert.test.ts" is failing intermittantly. I'm going to continue blithley forth, as if it is a problem with the test instead of my setup.

I suspect it has something to do with the amount of memory available, because the details of the test involve comparing the database to a snapshot, which seems.... memory intensive. I increased my rented VM's memory to 32GB and now it only fails intermittantly instead of consistently. That's good enough for me, but I was wondering if you want reports of unreliable tests?

**Comment by OracPrime** [comment]
Is there a minimum recommended RAM for development?  I'm finding the tests are killing my 16GB Ubuntu setup

**Comment by ghost** [comment]
Hello. I spent some time putting together a guide on how to setup Ubuntu from scratch to achieve successful `make test` 

See ticket #3148
